app/hooks/use-display-currency.ts

```diff
@@ -0,0 +1,10 @@
+import { useState } from "react"
+
+export const useDisplayCurrency = () => {
+  const [displayCurrency, setDisplayCurrency] = useState("USD")
```

Comment:
It doesn't appear this hook is used, do you have future plans for it? Would you add additional functionality to it because as it stands now it seems to be an unnecessary wrapper of useState.

Reply:
The updated version of this hook now has additional features, but it still relies on the LocalizationContext. Once we receive the display currency from the backend, we can replace the Context with the Apollo client and the hook should function properly.